include/picongpu/particles/ionization/byCollision/ThomasFermi/AlgorithmThomasFermi.hpp

```diff
@@ -100,7 +102,7 @@ namespace picongpu
 
                     float_64 const B = -math::exp(
                         thomasFermi::TFB0 + thomasFermi::TFB1 * T_F
-                        + thomasFermi::TFB2 * math::pow(T_F, float_64(7.)));
+                        + thomasFermi::TFB2 * math::pow<float_64, uint8_t>(T_F, 7u));
```

Comment:
There is no need to set `<float_64, uint8_t>`

Reply:
please use `pmacc::math::pow` else pow of alpak is called because `math::` points in picongpu to `alpaka::math`

Reply:
done